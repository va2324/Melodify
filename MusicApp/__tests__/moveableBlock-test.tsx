import React from 'react';
import renderer from 'react-test-renderer';
import MovableBlock from '@/components/movableBlock';

describe('moving block', () => {
    test('renders successfully', () => {
        const tree = renderer.create(<MovableBlock />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});