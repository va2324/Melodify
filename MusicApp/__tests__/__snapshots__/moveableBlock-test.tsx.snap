// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`moving block renders successfully 1`] = `
<View
  collapsable={false}
  onMoveShouldSetResponder={[Function]}
  onMoveShouldSetResponderCapture={[Function]}
  onResponderEnd={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderReject={[Function]}
  onResponderRelease={[Function]}
  onResponderStart={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  onStartShouldSetResponderCapture={[Function]}
  style={
    {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
      "transform": [
        {
          "translateX": 0,
        },
        {
          "translateY": 0,
        },
      ],
    }
  }
/>
`;
